test/import/FSHImporter.test.ts

```diff
@@ -76,6 +76,31 @@ describe('FSHImporter', () => {
     assertFixedValueRule(profile.rules[0], 'code', expectedCode);
   });
 
+  it('should parse escaped hash and backslash characters in system identifiers in codings', () => {
+    const input = `
+    Profile: HashBrowns
+    Parent: Observation
+    * code = https://breakfast.com/good\\\\food\\#potatoes#hash#browns
+    * extraCode = https://lastly.com/backslash\\\\#last
+    * bonusCode = \\\\\\\\#just_backslash "Just Backslash"
+    `;
+    const result = importSingleText(input, 'HashBrowns.fsh');
+    const profile = result.profiles.get('HashBrowns');
+    const expectedCode = new FshCode('hash#browns', 'https://breakfast.com/good\\food#potatoes')
```

Comment:
It is admittedly a little confusing, since these strings are already using `\` as an escape character. So the correct text to paste into a FSH file for this example would be:
`* code = https://breakfast.com/good\\food\#potatoes#hash#browns`
After parsing, this puts a literal `\` between `good` and `food`, and a literal `#` between `food` and `potatoes`.

But, to represent this text in a string, all the backslashes have to get escaped in order to correctly represent the FSH text. Does that clear things up?